portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,76 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
```

Comment:
If you want to use these, you will need to add a dependency in maven :) https://github.com/atreil/software-product-sprint-or-my-portfolio/blob/master/walkthroughs/week-2-server/step-5-datastore-walkthrough.md#maven-dependency